gradle.properties

```diff
@@ -1,3 +1,4 @@
 kotlinJvm=1.5.21
 jvmTarget=1.8
-version=1.0-SNAPSHOT
+version=0.1-SNAPSHOT
+nimbusJoseJWTVersion=9.11.3-SNAPSHOT
```

Comment:
Do you need to update gradle.properties here? I believe `nimbus` is still not used